lib/ObjWriter/objwriter.cpp

```diff
@@ -13,838 +13,715 @@
 ///
 //===----------------------------------------------------------------------===//
 
-#include "llvm/CodeGen/AsmPrinter.h"
```

Comment:
Generally it's better to include files where they're needed. I would imagine most of these are needed in the implementation c++ file, not the header.